.github/workflows/swift-toolchain.yml

```diff
@@ -238,8 +238,11 @@ jobs:
           components: 'Microsoft.VisualStudio.Component.VC.Tools.x86.x64;Microsoft.VisualStudio.Component.VC.Tools.ARM64'
           arch: ${{ matrix.arch }}
 
+      # TODO: Revert back to hendrikmuhs/ccache-action.
```

Comment:
Revert back to what it currently is? 🤔

Reply:
Whoops, missed some stale TODOs. thanks for catching :P